### PR TITLE
[FEATURE] Progress bar

### PR DESCRIPTION
**Pull request by martonlederer**
This is a very simple progress bar implementation. We will be able to implement this much more professionally with the new yolk API, but until then, this will work.

### CODE REVIEW

File: src/deps.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import ProgressBar from "https://x.nest.land/progress@1.1.0/mod.ts";
```
Comment (notfilippo) [inline src/deps.ts]: you can simply replace `import` with `export`

File: src/deps.ts [hunk 2]
```diff
@@ -11,3 +13,4 @@ export { installUpdateHandler, globalModulesConfigPath } from "https://x.nest.la
 export { getLatestVersionFromNestRegistry, getLatestVersion, analyzeURL, versionSubstitute } from "https://x.nest.land/eggs-update-handler@0.5.1/lib/registries.ts";
 export { version } from "./version.ts";
 export const lstatSync = Deno.lstatSync;
+export { ProgressBar };
```
Comment (notfilippo) [inline src/deps.ts]: ... and remove this

File: src/deps.ts (hunk 1 above)
Comment (martonlederer) [reply src/deps.ts]: i know, but thats not working idk why
let me double check

**Comment by notfilippo** [comment]
LGTM

**Comment by martonlederer** [comment]
Thanks for fixing the deps 😅

**Pull request merged**